sagemaker-pipelines/step-decorator/local-mode-example/using_step_decorator_with_local_mode.ipynb

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
<ol><li>Same above. The title level "Setup configuration file&nbsp;" is not correct.</li><li><br></li></ol><blockquote>you must add a SageMaker execution role ARN to <a href="https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4507/config.yaml" target="_blank">config.yaml</a>. You must also specify the execution role ARN with the <code>role</code> variable below.</blockquote><p>We don't need to take both of the actions, i.e. add role to config file and specify the role variable. Either one works.</p><p>However, as configuring role in the config file requires the specific role ARN composed of account ID, in the demo notebook, we can inform users about the config file solution and pick the role variable approach as you've done below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4507/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>